frontend/components/site/UserActionsTable.jsx

```diff
@@ -66,7 +62,11 @@ class UserActionsTable extends React.Component {
   }
 }
 
-UserActionsTable.propTypes = propTypes;
+UserActionsTable.propTypes = {
```

Comment:
I think the general pattern we have been following is to declare all the prop types up at the top of the file, that way its really apparent what input the component accepts.
.
If that seems weird we can start moving them to the bottom, but in general I find the current setup a bit more readable.

Reply:
I agree it would be good to be consistent. This one had part at the top/part at the bottom, so I just moved them both to where I was currently looking (the bottom). I'll file an issue to make these declarations consistent.

Reply:
https://github.com/18F/federalist/issues/1636